iron-flex-layout.html

```diff
@@ -259,6 +265,43 @@
       align-self: stretch;
     };
 
+    --layout-self-baseline: {
+      -ms-align-self: baseline;
+      /* currently not supported in safari */
+      -webkit-align-self: baseline;
+      align-self: baseline;
+    };
+
+    /* multi-line alignment in main axis */
+
+    /* currently not supported in ie */
+    /* currently not supported in safari */
```

Comment:
I don't think we have (intentional) instances in the code where something simply does not work when used (and I'd be interested for examples, so that we can fix them). In any case, I think it is bad to add something that we know is definitely not supported in 50% of the browsers.